gateway/1.4/modules/ROOT/pages/index.adoc

```diff
@@ -1,91 +1,151 @@
+=== Control plane
 
-Overall, Mule Gateway allows you to add a dedicated orchestration layer on top of your backend APIs and services to help you separate orchestration from implementation concerns. You can leverage the governance capabilities of API Manager to apply, among other capabilities, throttling, security, caching, and logging to your APIs.
+The control plane is a MuleSoft-hosted platform that centralizes management and observability, enabling you to:
 
-If you want to apply policies and generate analytics info, you must use either:
+* Build APIs.
+* Add policies.
+* Deploy APIs into the runtime.
+* Monitor APIs in Anypoint API Manager.
+* Configure the runtime in Anypoint Runtime Manager.
 
-* An API defined in a Mule application.
-* A Mule application that proxies your existing backend implementation.
+=== Runtime
 
-For more information, see xref:mule-gateway::mule-gateway-capabilities-mule4.adoc[Mule Gateway Capabilities].
+The gateway runtime is a customer-hosted distributed entity installed in any cloud data center or, for testing purposes, on your local laptop. The runtime receives commands from the control plane and routes and protects the backend APIs. Built with security in mind, communications between the runtime and the control plane are secured through mTLS and HTTPs.
+
+FluentBit implementation enables log output to local files, or to aggregators such as New Relic, Sumo Logic, Splunk, among others.
+
+== Replicas
+A runtime unit is called a `replica`. While a single runtime can support multiple backend APIs for high availability, MuleSoft recommends installing Flex Gateway as a cluster - a Kubernetes Ingress controller with multiple replicas running in parallel. Doing so increases gateway performance and robustness, and is recommended for high-performance applications that need to scale.
+
+An optional external REDIS storage can be configured for features such as distributed rate limiting and distributed caching. Gateway replicas use the REDIS storage as a scratchpad to coordinate rate-limiting across multiple replicas, so customer SLAs are adhered to.
+
+== Connected Mode vs. Local Mode
+
+Flex Gateway can be deployed in two ways:
+
+* Connected Mode
+* Local Mode
+
+=== Connected Mode
+In Connected Mode, the gateway is fully connected to the MuleSoft control plane. This allows for centralized management, observability, and security. Anypoint API Manager enables full API lifecycle management and policy configuration. Anypoint Runtime Manager enables you to deploy and configure your gateway.
+
+Choose Connected Mode for a UI-based experience to deploy policies, and for managing and monitoring the gateway.
+
+=== Local Mode
+
+You can also configure and manage a standalone gateway that is mostly disconnected from the control plane. Choose Local Mode for this experience.
+
+[NOTE]
+====
```

Comment:
I don't think you need the `====` if the Note is only one line.

Reply:
Fixed